### ISSUE TITLE
Documentation: --copy-links may cause huge backups

### ISSUE THREAD

**Issue by aryoda** [bit-team/backintime#1652]
The use of --copy-links should be mentioned in our FAQ as a reason for huge backups (if the backups source folder contains one or multiple links to target folder with large disk space consumption).

Also  a clear warning in the GUI could be helpful too (as tooltip or in the checkbox label):

![image](https://github.com/bit-team/backintime/assets/11374410/bba6580b-d9c6-4095-a347-bca1372a6561)

**Comment by emtiu** [comment]
I agree that **"Copy links (dereference symbolic links)"** is a confusing description. I think **"Follow symbolic links"** would be much clearer.

We might even consider spelling out the literal rsync options that our settings include. In that case, this checkbox (and other, accordingly) might be labelled: **"`--copy-links`: Follow symbolic links"**

I'm also fine with an FAQ entry to warn of including symlinks to large folders with this option.

But I don't think we need to issue an extra warning in the GUI. If people click around in the "Expert options" to activate an option regarding symbolic links, **and** they have symbolic links to huge folders in their included paths, then I think they can be expected to understand the consequences.

Or am I out of touch? ;)

**Comment by buhtz** [comment]
FAQ is good idea.

> I agree that **"Copy links (dereference symbolic links)"** is a confusing description.
> ...
> **"`--copy-links`: Follow symbolic links"**

I like that idea.

Brings me again to a big redesign of the settings dialog. ;)
Maybe "Follow symbolic links" is a good start to prevent an exploding PR. :rofl: 

> Or am I out of touch? ;)

No. Especially people clicking around in **Expert** Options do operate on their own risk. ;)

**Comment by aryoda** [comment]
Totally fine for me (nothing to add :wink:)

**Comment by buhtz** [comment]
<img width="608" height="274" alt="Image" src="https://github.com/user-attachments/assets/ecbae546-6d02-40f1-805b-387f7e192377" />

I think I'll rephrase the checkbox and its tooltip somehow.

**Comment by buhtz** [comment]
Please see PR #2347 where I replaced the two symlink checkboxes with a new widget in hope it is easier to understand for users.

![Peek 2026-01-06 16-42](https://github.com/user-attachments/assets/f633f630-32b4-438a-aba4-3d98eb14cb8a)
